src/agent.cc

```diff
@@ -149,9 +141,7 @@ void Agent::Build(Agent* parent) {
   this->AddToTable();
 }
 
-void Agent::EnterNotify() {
-  ctx_->RegisterAgent(this);
-}
+void Agent::EnterNotify() { ctx_->RegisterAgent(this); }
```

Comment:
I think our style guide is either intentionally quiet on this topic and/or prefers the original??
https://google.github.io/styleguide/cppguide.html#Conditionals